canonical-web-and-design/ubuntu.com
# Add edge month takeover and engage pages

## Done

- Add `Edge month` takeover
- Add `Edge month` landing page
- Replace `Ai webinar` takeover with `Edge month`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and [/engage/edge-month](http://0.0.0.0:8001/engage/edge-month)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to home page and refresh until you see the `Edge month` takeover 
- Compare to the [brief](https://docs.google.com/spreadsheets/d/1kunh8_7QioTrRwKR5Of9foxMTR1m_o_JCMrBgHgt-0A) and [design](https://github.com/canonical-web-and-design/web-squad/issues/1334)
- Go to [/engage/edge-month](http://0.0.0.0:8001/engage/edge-month) and compare with [brief](https://docs.google.com/spreadsheets/d/1kunh8_7QioTrRwKR5Of9foxMTR1m_o_JCMrBgHgt-0A) and [design](https://github.com/canonical-web-and-design/web-squad/issues/1336)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1335 and https://github.com/canonical-web-and-design/web-squad/issues/1337
